lib/helpers.coffee

```diff
@@ -1,8 +1,36 @@
 SelectorLinter = require 'atom-selector-linter'
+CSON = require 'season'
+fs = require 'fs-plus'
 
 exports.getSelectorDeprecations = ->
   linter = new SelectorLinter(maxPerPackage: 50)
   linter.checkPackage(pkg) for pkg in atom.packages.getLoadedPackages()
+  
+  userKeymapPath = atom.keymaps.getUserKeymapPath()
+  
+  if fs.isFileSync(userKeymapPath)
+    try
+      userKeymapCson = CSON.readFileSync(userKeymapPath)
+    catch error
+      console.warn("Failed to load keymap file: #{userKeymapPath}", error)
```

Comment:
I think you can :fire: this, core should be the one warning about this if anywhere.